### ISSUE TITLE
Scraper Check 11/00/2018

### ISSUE THREAD

**Issue by jewbmx** [I-A-C/script.module.lambdascrapers#35]
Scraper Check 11-08-2018
(Double Check My Findings!)
(Tested by hand in browser and 1by1 in my Scrubs addon.)
(Droid s7 edge  iowa, usa)

_enDbird
- tvbmoviez.py # Needs removed, Broken dupe for bestmovies.

_DbirdBroke
- phazeddl.py # Sites dead and tinydl.com doesnt work.
- power.py # Doesnt work for me, might be me but dont care to look at it.
- sceper.py # Sites dead doubt its ever coming back.
- wrzcraft.py # Sites 'Down for Maintenance' but says they moved to http://new.wrzcraft.net .
(Think i tested the new.wrzcraft.net and still was broke.)

_Dbird2Fix
- ddlvalley.py # Site works fine.
- gomovies.py # ddos/cf  cockblock
- rlsscn.py # Site works fine. (has another domain too http://tvdownload.net)
- scenerls.py # ddos/cf  cockblock
- seriescr.py # Site works fine.
(These can most likely be fixed maybe, Im not gonna tho.)

_DbirdFixed
- bestmoviez.py # fixed with cfscrape.
- moviesonline.py # Added new domains i found.
(feel free to hunt for a alternate domain for bestmoviez.py)
(http://jewrepo.cf/JewRepo/4git/)
(Hosting these for about 1 week then they will be deleted.)

_DbirdWorked
- 2ddl.py
- 300mbfilms.py
- directdl.py
- myvideolink.py
- rlsbb.py
- ultrahdindir.py

**Comment by host505** [comment]
Well done mate. Mind doing pr to include your fixed scrapers in ls? Or you want me to do it for you?
Yoda team fixed a few on their last update (wrzcraft is one that comes to me right now) but I don't feel comfortable just copying them to ls. They did use our rlsbb fix thought - they actually used both fixes, domain prefix change and cfscrape import - no idea why.

**Comment by jewbmx** [comment]
NP was already checking mine after going thru my folder of broken ones lol.
You can do the pr I dont really know my way around git that much lmao.

Didnt know yoda still got updates, guess I need to check my repos.

**Comment by shellc0de69** [comment]
> 
> 
> Well done mate. Mind doing pr to include your fixed scrapers in ls? Or you want me to do it for you?
> Yoda team fixed a few on their last update (wrzcraft is one that comes to me right now) but I don't feel comfortable just copying them to ls. They did use our rlsbb fix thought - they actually used both fixes, domain prefix change and cfscrape import - no idea why.

Wtf r u talking about host505? That was not ur fix!!!!
That is how anybody would fix rlsbb, fyi i had mine done on sunday. You homos were scratching ur heads on tuesday still. Don't lie and say u did it. 
You are lucky to have this jew lol

**Comment by shellc0de69** [comment]
truth is most of u guys need to go back to reddit where u belong newbs@addons4kodi lol
The only 2 other people that know what they are talking about is jewman and doko.
Now these other newbs are getting big heads like they know how to fix shit lol

**Comment by host505** [comment]
Whoa relax dude, sure there's just a couple ways to fix. I only saw the update you did yesterday how the hell should I suppose to know you did it earlier. Relax nobody tried to take credit for changing site prefix from you, but I sure didn't see your fix before applying mine.
And yes, we're lucky to have some people fixing stuff, what's your point?

**Comment by jewbmx** [comment]
Quick 10min Check of the new yoda update.
heres all the ones that dont find results for me.
maybe they will see this lol.

-bob.py is still broken for me.
-still sportin alluc too.
-missing the settings for the pron account info which makes pron.py useless if it still works.

- 1080P.py
- 123moviesc.py
- 123netflix.py
- 1movie.py
- 321movies.py
- RLSB.py
- afdah.py
- allrls.py
- bestmoviez.py
- bmoviez.py
- cinemamega.py
- cmovies.py
- cooltv.py
- ddls.py
- ddlvalley.py
- dizigold.py
- dltube.py
- einthusan.py
- filmfrantic.py
- filmxy.py
- flenix.py
- flixanity.py
- fmovied6.py
- freemovie6.py
- genvideos.py
- gogoanime.py
- gomovies.py
- hd.py
- hdmega.py
- hulufilm.py
- huluplus.py
- huluvip.py
- icefilms.py
- icouchtuner.py
- imdark.py
- kingmovies.py
- l23movies.py
- m4u.py
- m4ufree.py
- miradetodo.py
- movie4kis.py
- movie4kto.py
- movie4uch.py
- movie4ufree.py
- moviesgold.py
- moviesonline.py
- mywatchseries.py
- mzmovies.py
- netflixlogin.py
- onlinedizi.py
- phazeddl.py
- power.py
- primewire.py
- projectfree.py
- rajahoki88.py
- releasebb.py
- rlsmovies.py
- scenerls.py
- sceper.py
- seriescr.py
- sezonlukdizi.py
- showbox.py
- solarmovie.py
- sunmovies.py
- tvrelease.py
- vioozfree.py
- vodly.py
- watch-free.py
- watch32.py
- wnmnt.py
- xmovies.py
- xwatchseries.py

**Comment by jewbmx** [comment]
Woe didnt see your guys posts before posting mine lmao just copied it out my notes and tossed it in. Yall should smoke a bowl and eat a cupcake, stress is very bad :)

**Comment by shellc0de69** [comment]
> 
> 
> Whoa relax dude, sure there's just a couple ways to fix. I only saw the update you did yesterday how the hell should I suppose to know you did it earlier. Relax nobody tried to take credit for changing site prefix from you, but I sure didn't see your fix before applying mine.
> And yes, we're lucky to have some people fixing stuff, what's your point?

lol for one im not supremacy and my addon is not yoda. Truth is if u want to continue using .ru domain only way to fix is by adding cfscrape.......the only way dude. This is common knowledge in our community.
You are a newb with a big head now lol.
Dude u just DID try to take credit for the fix, read ur own post!!!!
Point is...ur a newb! Shut ur mouth, unless u know what ur talking about.

**Comment by shellc0de69** [comment]
I pretty sure were jew got the fix from......and he hates u just as much as i do lol

**Comment by shellc0de69** [comment]
> 
> 
> Woe didnt see your guys posts before posting mine lmao just copied it out my notes and tossed it in. Yall should smoke a bowl and eat a cupcake, stress is very bad :)

lol sry man, that shit makes me anti social lol. I will have a drink though :)

**Comment by host505** [comment]
You're all messed up. I didn't use the cfscrape fix. I used the .to domain that has not yet implemented cloudflare.
Jewbmx's solution was to utilize cfscrape on .ru domain.
Both ways work.
Yoda implemented both fixes: .to prefix and cfscrape, and I wondered why, because .to doesn't use cloudflare.
As for your insults, I won't be bothered kiddo, you obviously have issues.
And yes, I'm newb to coding, but I've been around long enough to don't care what dipshits like you say.

**Comment by shellc0de69** [comment]
ur 

> 
> 
> You're all messed up. I didn't use the cfscrape fix. I used the .to domain that has not yet implemented cloudflare.
> Jewbmx's solution was to utilize cfscrape on .ru domain.
> Both ways work.
> Yoda implemented both fixes: .to prefix and cfscrape, and I wondered why, because .to doesn't use cloudflare.
> As for your insults, I won't be bothered kiddo, you obviously have issues.
> And yes, I'm newb to coding (never claimed otherwise) but I've been around long enough to don't care what dipshits like you say.

ur not special. we all know about the .to from their twitter page. Im sure they did it for back up. who the hell wants to fix it twice?
Funny thing is u guys are so obsolete. One scraper fix in like 3 months lol. Your module is still full of dups and broken/old scrapers. Good luck newbs......might as well go back to addons4kodi lol

**Comment by shellc0de69** [comment]
> 
> 
> Woe didnt see your guys posts before posting mine lmao just copied it out my notes and tossed it in. Yall should smoke a bowl and eat a cupcake, stress is very bad :)
> 
> I keep slacking and forget to tell everyone that you can feel free to jack the scrapers out of Scrubs, i need to upload my update still but they are usually always all working and universal since they are from all sorts of exo/cov addons lol. Already know my next update is gonna get jacked by plenty of people anyways due to what im droppin.

Im pretty sure what ur u going to be dropping and where u jacked it from....just saying ;)

**Comment by host505** [comment]
Like I said, you obviously have issues.
When did I implied I was special because I changed a fuckin domain prefix?
Always a drama bitch as you always were.

**Comment by jewbmx** [comment]
You still talkin to that troll account host505 lmao dipshit cant even hide right

Google found you hater

Rick Irvin - YouTube
https://www.youtube.com › shellc0de69
EA Suffers

**Comment by shellc0de69** [comment]
> 
> 
> Like I said, you obviously have issues.
> When did I implied I was special because I changed a fuckin domain prefix?
> Always a drama bitch as you always were.

No im just right, and u can't except it lol. Yes u again think u and this fake fucking team magically fixed rlsbb..when in fact u didnt do shit. 
Now u think u know who i am lol. wrong lol

**Comment by host505** [comment]
@jewbmx he actually doesn't want to hide (much), he's too big of an attention whore to do so ;)

**Comment by shellc0de69** [comment]
> 
> 
> You still talkin to that troll account host505 lmao dipshit cant even hide right
> 
> Google found you hater
> 
> Rick Irvin - YouTube
> https://www.youtube.com › shellc0de69
> EA Suffers

wow u hit the jackpot buddy good job lol

**Comment by host505** [comment]
First of all, ls & ex redux didn't start as a 'team' project, ie the name wasn't decided by a team.
Heck, I personally don't consider myself as part of any team, if I have to offer something I will.
Personally I don't care much about ex redux, I use ls on my own add-on.
And I don't even like the name also - I don't think λ would too, he would probably want his name to be vanished, after all the size that what he created became.

But what you're always doing is belittling every newb attempt. I didn't see any of the major devs that created all these to come at you when you created your monkey business.

**Comment by jewbmx** [comment]
Yall are fucking up whats supposed to be a nice cleaning post with all this childish bickerin, take it elsewhere so people can actually see the reason this post is here.

**Comment by shellc0de69** [comment]
> 
> 
> Yall are fucking up whats supposed to be a nice cleaning post with all this childish bickerin, take it elsewhere so people can actually see the reason this post is here.

Thats funny coming from u, cause they r telling u that daily lol

**Comment by shellc0de69** [comment]
Why even try to put lambda's name in this shit hole of a scraper module that never gets fixed. Man, if i was him i would be fucking pissed lol

**Comment by shellc0de69** [comment]
Sure, let's copy and paste scrapers that other real devs have worked on, and name it Lambda scrapers. what a fucking disgrace

**Comment by jewbmx** [comment]
Yea i sort my scrapers a little different in my addon and placed moviesonline in my messy section lol it probably needs some tlc. Bestmovies i consider to be a good 1. I dont really like scrapers that pull more then 5 or 6 results.

**Comment by bjgood** [comment]
shellc0de69,
Why don't you just go away, dude. At least host505 and jewbmx are contributing. They're coming up with some pretty good code...that works. As one of them said...you're just a troll. You're, obviously, not a nice person.

Like I said, why don't you just go away. We like "positive" attitudes around here.

**Comment by SerpentDrago** [comment]
@I-A-C  , can you please delete / block / remove shellc0de69's messages , he is causeing a mess in a great thread about trying to fix these scrapers . (this is the same dude that caused issues on a4k . Pretty sure its blamo lol )

Anyways , Great work guys  @jewbmx  Awesome job looking into all those scrapers thats alot of time !

**Comment by SerpentDrago** [comment]
@jewbmx  All these changes are in your addon  ? can i get a link ? 

@host505  I think you understand more then i do whats going on here , wanna do the PR , I guess we should put non working ones in a folder ? and not just delete ?

**Comment by SerpentDrago** [comment]
We need a built in scraper Checker . that auto disables scrapers not working .  !

**Comment by jewbmx** [comment]
I most likely already have a few that are fixed in my addon but not all of em lol. My phones folder has a bit more done just need to finish my fix folder too lmao

**Comment by jewbmx** [comment]
You can get my stuff from clickin my profile on here and then the reg 'repo'

**Comment by jewbmx** [comment]
Use the icefilms style code that has hostchecker at the top lol fails to local host if i recall

**Comment by SerpentDrago** [comment]
> Use the icefilms style code that has hostchecker at the top lol fails to local host if i recall.
> 
> I just do a full check every month if not weekly.

what do you mean by this ? 


also the zip in your repo says its 3 days old . Was not sure if that was updated . its just zips

**Comment by host505** [comment]
> @host505 I think you understand more then i do whats going on here , wanna do the PR , I guess we should put non working ones in a folder ? and not just delete ?

Removing them from [this list](https://github.com/I-A-C/script.module.lambdascrapers/blob/master/lib/default.py#L81-L89) should be sufficient.
Tbh I find the current architecture/settings set a little confusing. There is a 'default for current scraper' setting that only enables the aforementioned set, and then there's 'enable all', that enables all in the current sources folder? There are many duplicates/broken ones in there. They're not enabled by default, but user shouldn't have the ability to enable them all, even manually.
So yes, separating broken/duplicates and ditching some settings is needed.

**Comment by supremacy-addon** [comment]
how about u take yoda scrapers out

**Comment by I-A-C** [comment]
Gone!!

**Comment by supremacy-addon** [comment]
cool

**Comment by I-A-C** [comment]
@host505,
I  just pushed a commit aligning the defaults to jewbmx's list.  Now that I've dropped Yoda, I'm going through the settings.xml to remove their now obsolete scrapers from lambdascrapers.

I acknowledge the current architecture/settings is a bit confusing... too many options I guess.  Just because you CAN 'enable all providers' doesn't necessarily mean that you SHOULD enable all.  I'm going to remove the enable/disable providers lines from 'Modules' section.  I do find the enable/disable all helpful in the 'foreign' and 'debrid' section and to a lessor degree, the  'providers' section also

**Comment by host505** [comment]
> Just because you CAN 'enable all providers' doesn't necessarily mean that you SHOULD enable all

Yes but users will do so.
To be clear because it isn't to me: does this setting ('enable all providers for current module') suppose to enable all providers within the selected module's folder?
And what's the difference (if any) with the Providers tab/Enable all providers?
If these settings enable all providers within a source's folder, imo we should either clean the lambdascrapers/en folder (leave only what is in the 'default' list), or ditch these settings for this module.
Shouldn't allow the user to enable the duplicates in any case.

**Comment by I-A-C** [comment]
> To be clear because it isn't to me: does this setting ('enable all providers for current module') suppose to enable all providers within the selected module's folder? 

Yes

>  And what's the difference (if any) with the Providers tab/Enable all providers?

This tab enables all Providers independent of module;  meaning all of the Placenta, Incursion,(previously Yoda) scrapers are enabled.

I have a commit ready that would clean out the nonworking scrapers from the lambdascrapers/en folder.
I just want to get setting.xml file cleaned up before I push that.

**Comment by jewbmx** [comment]
Id just remove all the old addon folders, i find em useful but dont wanna be checking all of those while also maintaining the main pile. Also for any broken ones you can just move them to a folder labeled anything ya want like _Down and then we can easily find em to fix later. Settings can stay since the item isnt in the right spot to be activated. Same for dupes _Spares but leave em out the settings and worry about them when they are needed

**Comment by jewbmx** [comment]
Heres a small portion of the nojoys

- 1080P.py # fixed this a while back lol thanks to Muad'Dib's youtube stream :) <3 that Kodi Guru.
- 123moviesgold.py # dead
- 123netflix.py # dead
- allrls.py # dead
- bmoviez.py # pretty sure this is a broken dupe of bestmoviez.py maybe try to remove the dbird lol.
- bob.py # this is broken even tho it shows a result, needs a free account login upon play.
(I switched to bobmovies.site aka mybobmovies.py)
(If ya wanna mess with the old bob.py you can use  user:jewtest pass:jewtest)
- icouchtuner.py # couchtuner closed shop but can use mycouchtuner.py the knockoff.
- l23movies.py # fixed
- mywatchseries.py # fixed
- projectfree.py # this ones coded for movies but dont work.
(I prefer the one coded for tv shows (myprojectfreetv.py))
- pron.py # might still work, its code is account based but not messing with a porn site.
- solarmoviez.py # fixed

**Comment by redditreaper** [comment]
┳┻| _
┻┳| •.•) Daddy, Are Copy and Pasters Gone Yet?
┳┻|⊂ﾉ
┻┳|
/﹋\
(҂`_´) -NOPE, THEY'RE GETTING MORE RETARDED EVERYDAY 
<,︻╦╤─ ҉ ҉ - - - - - - - - - 
/﹋\

**Comment by jewbmx** [comment]
Cant help myself lol does anyone else see how stupid it is to copy & paste a lame lil ascii picture making fun of copy & pasters? Callin the kettle black 🖕

**Comment by SerpentDrago** [comment]
@jewbmx  where do i get your latest fix's changes ? i like to be a bit more updated and bleeding edge then waiting on changes to the official repo for lambada (this )

**Comment by jewbmx** [comment]
Scroll to the start of this post lol my bad i forgot to say where i put em.

**Comment by jewbmx** [comment]
iwatchflix - now broken lol was too slow on the warning.

darewatch - reported down by email, dont use it but looks like a ddos protection issue with cf

**Comment by jewbmx** [comment]
Pretty sure about 5 or more other scrapers needed fixed today but i didnt keep track of em also dont remember if ya even use em in this project.

**Comment by jewbmx** [comment]
Im starting to think we should leave this open and redo it every week or month so people can use it to point out scrapers that might need looked at too. Like right now if you were clicking away and seen that a usual scraper result isnt there it prob needs checked lol, or you can see the same 4 scrapers are always the last to finish, they prob need checked too even tho they could just be slow ones

**Comment by doko-desuka** [comment]
> how about u take yoda scrapers out

@supremacy-addon I don't think that's fair of you, you have the primewire scraper from LS, you only removed the comments (you forgot one btw, "# In milliseconds."), and also the darewatch one.
What are you protecting? What's the end goal? I don't think anybody really cares who's the author, they just care that it works.  
I wouldn't mind one bit if all these addons shared each other's scrapers. We're stronger together and that sort of thing...  

@jewbmx nice catch w/ darewatch, they added a cloudflare anti-DDoS step.  
UniversalScrapers has a nice [CF challenge solver](https://github.com/I-A-C/script.module.lambdascrapers/blob/a6c365c3fb9b69ac116e8c7e2840b4597141ba27/lib/lambdascrapers/sources_%20lambdascrapers/en/plocker.py#L421-L599), but I think Exodus also has one, the modules/cfdecoder.py

**Comment by SerpentDrago** [comment]
To be fair to supremacy , and Doko i agree with you by the way .  I think having yoda as a MODULE to select is the major issue , not the fact that scrapers are mixed / whatever .  It confuses users , who knows mybee they go complain to sup about yoda issues and they are just useing yoda modules in this addon . 

I give the benefit of the doubt prob. but whatever 

IMHO Lambda scrapers should be one module , All mashed together and regularly updated (basically whats starting to happen )

**Comment by doko-desuka** [comment]
That makes sense Serpent, I hope you're right.

**Comment by jewbmx** [comment]
I think the request was to remove the yoda collection, dont think the singles in the main folder mattered. And if they got removed just get new ones from eggman's Poached, Numbers, or even my addon lol. I personally dont think anyone should complain about a copied scraper especially now that everyone keeps copyin that boxset  code that i think numbers made by hand lol

**Comment by host505** [comment]
Hey, @jewbmx , do you have any other goods in your basket?
I've been tested some of your fixed scrapers, I've cherrypicked some and want to make a pr with them on ls (if you don't mind).
If you have some more, would you please upload them on the server from the first post?
I've also checked your add-on, picked from there some as well.

Side question, do you know where the 1080P scraper originated from? Also, the version on your add-on, was it fixed by you or copied from Yoda?
Thanks mate, keep'em coming!

**Comment by jewbmx** [comment]
Feel free to share em. As to 1080p its commented at the top lol fixed by me from Muad 'Dibs youtube video. The first copy came from placenta before the rewrite.

**Comment by jewbmx** [comment]
Make sure you compare em to scrapers named like em. People seem to be tossin my 'my' scrapers into their folders next to 3 of matching scrapers lol

**Comment by jewbmx** [comment]
Anyone having issues getting scrapers to work? I been getting emails saying that every scraper that uses the ip_player code in its resolve section is broken. If I recall theres around 8 or more that use that code. Also the error wont be noticed unless someone trys to play a source from one of these scrapers.

**Comment by doko-desuka** [comment]
The resolve( ) function on each scraper is only called when the user tries to play an item from the "PROVIDER | hostname" list after everything has been scraped.  
Some websites (called "providers") hide the final address to the host with the video behind a hashed request (like **http\:\/\/m.primewire.gr/go.php?id=1692865**, which returns something like **https\:\/\/openload.co/embed/Tt0V8lttcEI**).  
That's why the resolve( ) function is needed, so you can lazily ask the source website for a single item, the one that the user chose to play. This saves bandwidth.  

That being said, what the heck is ip_player, a simple search doesn't find it =(

**Comment by jewbmx** [comment]
The issue seems to be something with this... (havent bothered with it yet personally lol.)


    def resolve(self, url):
        try:
            urldata = urlparse.parse_qs(url)
            urldata = dict((i, urldata[i][0]) for i in urldata)
            post = {'ipplugins': 1,'ip_film': urldata['data-film'], 'ip_server': urldata['data-server'], 'ip_name': urldata['data-name'],'fix': "0"}
            p1 = client.request('http://0123movies.org/ip.file/swf/plugins/ipplugins.php', post=post, referer=urldata['url'], XHR=True)
            p1 = json.loads(p1)
            p2 = client.request('http://0123movies.org/ip.file/swf/ipplayer/ipplayer.php?u=%s&s=%s&n=0' %(p1['s'],urldata['data-server']))
            p2 = json.loads(p2)
            p3 = client.request('http://0123movies.org/ip.file/swf/ipplayer/api.php?hash=%s' %(p2['hash']))
            p3 = json.loads(p3)
            n = p3['status']
            if n == False:
                p2 = client.request('http://0123movies.org/ip.file/swf/ipplayer/ipplayer.php?u=%s&s=%s&n=1' %(p1['s'],urldata['data-server']))
                p2 = json.loads(p2)
            url =  "https:%s" %p2["data"].replace("\/","/")
            return url
        except:
            return

**Comment by doko-desuka** [comment]
Try this:
```
def resolve(self, url):
    try:
        urldata = urlparse.parse_qs(url)
        urldata = dict((i, urldata[i][0]) for i in urldata)
        post = {'ipplugins': 1,'ip_film': urldata['data-film'], 'ip_server': urldata['data-server'], 'ip_name': urldata['data-name'],'fix': "0"}
        p1 = client.request('http://0123movies.org/ip.file/swf/plugins/ipplugins.php', post=post, referer=urldata['url'], XHR=True)
        p1 = json.loads(p1)
        p2 = client.request('http://0123movies.org/ip.file/swf/ipplayer/ipplayer.php?u=%s&s=%s&n=0' %(p1['s'],urldata['data-server']))
        p2 = json.loads(p2)
        url =  "https:%s" %p2["data"].replace("\/","/")
        return url
    except:
        return
```
The api.php call seems to have been removed

**Comment by jewbmx** [comment]
NoJoy there. Before that change it listed some results that didnt play, now it doesnt find any results for anything.

**Comment by doko-desuka** [comment]
That function isn't called until you play an item. If it's not even finding any results then there's a syntax error preventing the script to be loaded at all.  
Check if the indentation is right, it usually messes up when it's copy-pasted

**Comment by jewbmx** [comment]
Ah man your gonna get me sucked in lol, im just chillin watching a christmas story and playing with playlists :p

**Comment by jewbmx** [comment]
Tried your fix on two different scrapers and no matter how i add the new code it seems to break the scraper and make it not show up at all in the search lol, original scraper still loads. Its a odd issue for sure.

**Comment by jewbmx** [comment]
Heres a heads up for everyone that uses multiple versions like this lol. Name is the website minus the .dot

Testing on ( Hunter Killer )
(can be 96 results by ditching the first bit.)

moviesonlinefm.py
98 results

moviesonlinegy.py
98 results

moviesonlinela.py
98 results

moviesonlinemn.py
98 results

moviesonlinemx.py
98 results

moviesonlinetl.py
98 results

**Comment by jewbmx** [comment]
Looks like a few of those scraper issues emailed to me yesterday were just issues with http/https changes.

Im gonna have to redownload a fresh copy of LS and go thru what ya got so your upto date as well.

The past two weeks have been pretty busy in the world of sites and/or scrapers, alot of random petty changes and a few domain swaps.

**Comment by jewbmx** [comment]
Just did a new PR with todays scraper check and fixed what needed fixin in the normal folder. Someone might wanna check it out, added the notes for it to the pr description

**Comment by jewbmx** [comment]
Got 3 questions for the "LS peeps"
1. Does anyone wanna test a few scrapers i put in a folder that are halfass working but dont play for me?(maybe even figure out whats wrong with em for me lol)

2. Why do i see 2 different nojoy folders in LS lol?

3. Which one of ya is eggman lmao im always getting new module updates after i update any scrapers so im guessing the pretty man is around here keeping up with us :)  (if ya are around then nice job keeping up and kudos to your progress.)

**Comment by doko-desuka** [comment]
What folder did you put the nonworking ones, exactly?

**Comment by jewbmx** [comment]
Oh i didnt move or alter anything. Besides deletein that dupe completely lol. I prefer to post notes and let you guys decide what to do with the information.

Both nojoy folders might have dupes inside that are the scrapers before their fix, not sure if i checked after i got done the other day.

Im currently going thru every single lang. folder and plan to add them to a PR for everybody when i get done in a day or two. Been driving myself insane with errors from this stupid rd setting i wanted so i havent made it very far yet with the holiday too lol.

**Comment by jewbmx** [comment]
I feel like this is a uber stupid question but do i need to make the language files for the addon before i upload these language scrapers? (Feels so trump right now)

**Comment by jewbmx** [comment]
Dont remember if i said so yet but im done with all the language scrapers and waiting to be told to upload em by the big boss lol. Still have a couple to test but their domains are up to date and just gotta find a file to try em with.

**Comment by jewbmx** [comment]
Man im about to throw my phone and quit coding lol i cant figure out why my artwork aka theme option dont work in k18 ExoRedux works and from what i can see my code all matches lol beards gonna go bald over here
